packages/osd-pm/README.md

```diff
@@ -141,6 +141,40 @@ yarn osd
 Bootstrapping also calls the `osd:bootstrap` script for every included project.
 This is intended for packages that need to be built/transpiled to be usable.
 
+#### Single-version validation
+
+Bootstrapping, by default, applies a `strict` single-version validation where it requires all the dependencies defined
```

Comment:
could we clarify what does single version validation mean and probably giving an example. that will help. let's say HelloPlugin

Reply:
This is really all that it is:
```
Bootstrapping, by default, applies a `strict` single-version validation where it requires all 
the dependencies defined more than once to have the same version-range in the `package.json`
files of Dashboards, packages, and plugins. If a violation is identified, bootstrapping terminates
with an error.
```

Unfortunately, I don't know the exact reasons behind its existence; I have some theories but I am having a hard time defending them.